apps/src/applab/applab.js

```diff
@@ -666,6 +666,9 @@ Applab.init = function(config) {
   studioApp.runButtonClick = this.runButtonClick.bind(this);
 
   Applab.channelId = config.channel;
+  if (config.level.testAssetPathPrefix) {
+    Applab.assetPathPrefix = config.level.testAssetPathPrefix;
+  }
```

Comment:
I don't like always setting assetPathPrefix to "//localhost:8001/..." because then we lose test coverage of the default "/v3/assets/" assets path prefix. `__TestInterface` seems to be used widely in our codebase already, so I'll try that approach.